languages/nim/exercises/concept/basics/.meta/example.nim

```diff
@@ -0,0 +1,19 @@
+const expectedMinutesInOven* = 40
```

Comment:
Is the `*` the "export identifiers" concept listed in the "Learning concepts"?

Reply:
Yes